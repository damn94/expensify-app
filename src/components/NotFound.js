import React from 'react';
import {Link} from 'react-router-dom'
//COMPONENT NOTFOUND
const NotFound = () => (
	<div>
		<h1>404 - <Link to='/'>go home</Link></h1>
	</div>
)



export default NotFound;