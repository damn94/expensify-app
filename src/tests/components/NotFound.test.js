import { shallow } from 'enzyme';
import React from 'react';
import NotFound from '../../components/NotFound'

//TEST for NotFound
test('Should render the NotFound component',()=>{
	const wrapper = shallow(<NotFound />)
	expect(wrapper).toMatchSnapshot();

})