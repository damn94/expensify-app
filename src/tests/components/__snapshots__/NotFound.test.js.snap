// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the NotFound component 1`] = `
<div>
  <h1>
    404 - 
    <Link
      replace={false}
      to="/"
    >
      go home
    </Link>
  </h1>
</div>
`;
